Plugins/Flow.Launcher.Plugin.Program/Views/Commands/ProgramSettingDisplay.cs

```diff
@@ -9,20 +9,11 @@ namespace Flow.Launcher.Plugin.Program.Views.Commands
 {
     internal static class ProgramSettingDisplay
     {
-        internal static List<ProgramSource> LoadProgramSources(this List<Settings.ProgramSource> programSources)
+        internal static List<ProgramSource> LoadProgramSources(this List<ProgramSource> programSources)
```

Comment:
I don't think we need to copy the programsource🤔. I would suggest whether it is possible to do a bit more refactor by using `UnionBy`, but it's ok to leave it like this.

Reply:
> I don't think we need to copy the programsource

Yep, seems like we don't need to do a deep copy.